flake.nix

```diff
@@ -0,0 +1,35 @@
+{
+  description = "The Zoekt developer environment Nix Flake";
+
+  inputs = { nixpkgs.url = "nixpkgs/nixos-unstable"; };
+
+  outputs = { self, nixpkgs }: {
+    devShells = nixpkgs.lib.genAttrs [
+      "x86_64-linux"
+      "aarch64-linux"
+      "aarch64-darwin"
+      "x86_64-darwin"
+    ] (system:
+      let
+        pkgs = import nixpkgs {
+          inherit system;
+          overlays = [ self.overlays.ctags ];
+        };
+      in { default = import ./shell.nix { inherit pkgs; }; });
+    # Pin a specific version of universal-ctags to the same version as in cmd/symbols/ctags-install-alpine.sh.
+    overlays.ctags = self: super: rec {
```

Comment:
You could add sg/sg flake as an input and use the overlay from there instead 😃

Reply:
:O :O maybe the go-ctags repo would be the better place to have the flake?

Reply:
That makes a lot of sense I think? 🤔